package.json

```diff
@@ -13,10 +13,10 @@
     "css": "node-sass ./output/site/styles/main.scss ./output/site/static/styles/main.css",
     "css:watch": "npm run css -- --watch",
     "css:build": "cd output && postcss ./site/static/styles/main.css --use autoprefixer --use cssnano -r",
-    "js": "concat -o ./output/site/static/scripts/main.js ./output/site/static/js/*.js",
-    "js:watch": "watch 'npm run js' ./output/site/static/js --wait=3",
+    "js": "concat -o ./output/site/static/js/rollup-target.js ./output/site/static/js/local.js ./output/site/static/js/main.js ./output/site/static/js/modernizr-custom.js; rollup -c",
```

Comment:
If you import the required files into the main target then you dont need to use concat.

Reply:
I think I understand what you are doing here and I don't like it. Perhaps the rollup-target js file should exsts somewhere else?

Reply:
I guess I could make a `temp` directory and output `rollup-target.js` in there (and add it to gitignore).  That will eliminate the need for the watch filter.  And the concat line can just revert to how it was before... concat is still required to combine `local.js` with the `main.js` and `modernizr-custom.js` files from core.

Sound okay?

Reply:
Using import statements from local files to core files should remove this requirement.

Reply:
Ah got it!